docs/questions/query-builder/expressions-list.md

```diff
@@ -535,6 +568,22 @@ Syntax: `replace(text, find, replace)`.
 
 Example: `replace([Title], "Enormous", "Gigantic")`.
 
+### splitPart
+
+> Only available on PostgreSQL.
+
+Splits a string on a specified delimiter and returns the nth substring.
+
+Syntax: `splitPart(text, delimiter, position)`.
+
+`text`: The column or text to return a portion of.
+
+`delimiter`: The pattern describing where each split should occur.
+
+`position`: Which substring to return after the split. Index starts at position 1.
+
+Example: `splitPart([Date string], " ", 1)`. If the value for `Date string` was `"2024-09-18 16:55:15.373733-07"`, `splitPart` would return `"2024-09-18"` because it split the data on space (`" "`, and took the first part (the substring at index 1)).
```

Comment:
I suggest using an example of something that is a text string, instead of something that looks like a date.
I would also suggest not using a space because that makes it difficult to parse. Something like splitting on commas, or "/" (like I do with URLs every day)